test/strongbox/gui_test.clj

```diff
@@ -1,6 +1,6 @@
 (ns strongbox.gui-test
   (:require
-   [clj-http.fake :refer [with-fake-routes-in-isolation]]
+   [clj-http.fake :refer [with-global-fake-routes-in-isolation]]
```

Comment:
replace all instances of `with-fake-routes-in-isolation` with `with-global-fake-routes-in-isolation`